.github/workflows/encoding.yml

```
@@ -11,3 +11,11 @@ jobs:
         run: |
           files=$(ls Northstar.Client/mod/resource/northstar_client_localisation_*.txt)
           IFS=$'\n'; files=($files); unset IFS; ! file --mime "${files[@]}" | grep -v "charset=utf-16le"
+  check-missing-translations:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```

v2 is uses NodeJS 12 which is deprecated.
(We still need to update one remaining use of it in our CI ^^)